modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartQuickAdd.js

```diff
@@ -117,6 +117,63 @@ export default function CartQuickAdd() {
 
 				const {name, productConfiguration, urls} = parentProduct;
 
+				if (product.discontinued) {
```

Comment:
what if the replacement is in a different index?

Reply:
Did you select the wrong line for this comment? @andrea-ale-sbarra